docs/source/usageexamples.rst

```diff
@@ -994,6 +994,73 @@ to be contiguous if an * .aem instead of a * .omx file) however.
     mat.load('demand.aem')
     mat.computational_view(['red_cars', 'blue_cars'])
 
+.. _turn_volumes:
+Extract turn volumes
+~~~~~~~~~
+The extraction of turning movements requires assignment paths. They have to be saved while
+assigning.
+Turn movements can be extracted from the assignment class once the assignment is executed or using an
+existing assignment results table.
+
+The turn volumes functions require a dataframe containing the a, b, and c nodes of the desired turns.
+
+To save turn movements from the assignment class it will be sufficient to use save_turning_volumes.
+
+
+::
+
+    # Create (or load) a dataframe with turns' a,b, c nodes
+    turn_abc = pd.DataFrame(
+        [
+            [1, 2, 3],
+            [4, 5, 6],
+        ],
+        columns=["a", "b", "c"]
+    )
+
+    # add a line to save turning volumes to the specified table
+    # the saving option also returns the dataframe containing turning volumes
+    turning_volumes = assigclass.save_turning_volumes("my_turn_results", turn_abc)
+
+    # to retrieve the turning volumes without saving to a table you can use this line instead
+    turning_volumes = assigclass.turning_volumes(turn_abc)
```

Comment:
I believe this is supposed to be on the assignment rather than the `TrafficClass`
```diff
-    turning_volumes = assigclass.save_turning_volumes("my_turn_results", turn_abc)
+    turning_volumes = assig.save_turning_volumes("my_turn_results", turn_abc)

    # to retrieve the turning volumes without saving to a table you can use this line instead
-    turning_volumes = assigclass.turning_volumes(turn_abc)
+    turning_volumes = assig.turning_volumes(turn_abc)
```